PopPUNK/__main__.py

```diff
@@ -166,6 +176,15 @@ def get_options():
                                               '[default = False]', default=False, action='store_true')
     queryingGroup.add_argument('--accessory-only', help='Use an accessory-distance only model for assigning queries '
                                               '[default = False]', default=False, action='store_true')
+    queryingGroup.add_argument('--assign-lineage', help='Assign to lineages rather than strains - can specify '
```

Comment:
I think this is --lineage-clustering + --assign-strain. Maybe should be its own mode

Reply:
Created --assign-lineages mode in effb8909e329c8ac57e1bc9aea09be1088f7021d.